src/components/searchBar.test.tsx

```diff
@@ -29,40 +29,52 @@ function localStorageMock() {
   };
 }
 
+const handleSearch = jest.fn();
+
 describe('SearchBar', () => {
   it('renders component', () => {
-    render(<SearchBar placeholder="Search test" />);
+    render(<SearchBar placeholder="Search test" searchHandler={handleSearch} />);
     expect(screen.getAllByRole('button').length).toBe(2);
     expect(screen.getByRole('textbox')).toBeInTheDocument();
     expect(screen.getByPlaceholderText('Search test')).toBeInTheDocument();
   });
-  it('changes input vakue by onCnange event', () => {
-    render(<SearchBar placeholder="Search test" />);
+  it('changes input value by onCnange event', () => {
+    render(<SearchBar placeholder="Search test" searchHandler={handleSearch} />);
     userEvent.type(screen.getByRole('textbox'), 'test search');
     expect(screen.getByDisplayValue('test search')).toBeInTheDocument();
   });
   it('clears input value by clear button onClick event', () => {
-    render(<SearchBar placeholder="Search test" />);
+    render(<SearchBar placeholder="Search test" searchHandler={handleSearch} />);
     userEvent.click(screen.getAllByRole('button')[1]);
```

Comment:
I would recommend adding uniq test identifiers or ids to these buttons instead of screen.getAllByRole('button')[1]. Reviewers do not have the same context in mind as you have